### DISCUSSION TITLE
[mpvnet] 小白关于简易版使用svp4补帧的问题

### DISCUSSION THREAD

**Discussion by MADAXIAO** [hooke007/mpv_PlayKit#199] [libmpv]
纯小白，想用svp4插件来补帧，请问是否是在官网下载svp4安装之后，在高级设置里面输入mpvpipe然后保持manager运行就行。我先试了破解版的svp4，安装之后在设置里面输入，然后重启没有反应，svp4客户端组件已经检查并安装了。请问怎么解决？

**Comment by hooke007** [comment]
https://github.com/hooke007/mpv.net_CM#%E5%85%BC%E5%AE%B9-svp-manager
我不知道破解版什么情况，正版按照这个流程是没有问题的

**Comment by MADAXIAO** [comment]
这样操作之后保持svp4在后台运行就可以补帧了吗，还是需要其他的操作？

**Reply by hooke007** [reply]
svp manager里选择使用的方案啊，它自带的播放器怎么用别的播放器就怎么用，除了前置步骤外没有区别。
关于svp manager的深入使用问题建议询问更专业的论坛。
